### PR TITLE
Various improvements

### PR DESCRIPTION
**Pull request by msksk**
line 711 (switch) and 721 (ps3)
Keiichi wanted to say Mion in the console version couldn't carry the model gun because of publisher's prior censorship.

line 3849 (switch)
中の人 is a voice actor(actress).

### CODE REVIEW

**Comment by msksk** [comment]
This is  about a comment line and doesn't matter to the script. So I just report it here. 3406th line (mio_omakn_switch.txt) doesn't match with the actual Japanese text.

File: Update/mio_omake_switch.txt [hunk 1]
```diff
@@ -127,7 +127,7 @@ void main()
 	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#5ec69a>詩音</color>", NULL, "<color=#5ec69a>Shion</color>", NULL, Line_ContinueAfterTyping); }
 	ModPlayVoiceLS(4, 6, "ps3/s20/06/440600185", 256, TRUE);
 	OutputLine(NULL, "まぁ、たくさん話に参加できたのに文句を言っちゃ、罰が当たるってもんですけどね」",
-			NULL, "Though I did spend most of them complaining despite playing a part, so I guess I got what I deserved.\"", Line_Normal);
+			NULL, "Though thankfully we had many chances to play a part, so we should not moan about it, you know.\"", Line_Normal);
```
Comment (DoctorDiablo) [inline Update/mio_omake_switch.txt:130]:
```suggestion
			NULL, "Though thankfully we had many chances to play a part, so we shouldn't moan about it, you know.\"", Line_Normal);
```

File: Update/mio_omake_switch.txt [hunk 2]
```diff
@@ -452,7 +452,7 @@ void main()
 	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#f0953d>レナ</color>", NULL, "<color=#f0953d>Rena</color>", NULL, Line_ContinueAfterTyping); }
 	ModPlayVoiceLS(4, 2, "ps3/s20/02/440200412", 256, TRUE);
 	OutputLine(NULL, "「……本当にそれ、自慢じゃないね」",
-			NULL, "\"...Should you really be bragging about that?\"", Line_Normal);
+			NULL, "\"...Indeed, it's really not a bragging.\"", Line_Normal);
```
Comment (DoctorDiablo) [inline Update/mio_omake_switch.txt:455]:
```suggestion
			NULL, "\"...Indeed, that really isn't bragging.\"", Line_Normal);
```

File: Update/mio_omake_switch.txt [hunk 3]
```diff
@@ -758,7 +758,7 @@ void main()
 			NULL, " Rena was the main heroine when \"Higurashi\" started, right?", Line_WaitForInput);
 	ModPlayVoiceLS(4, 2, "ps3/s20/02/440200418", 256, TRUE);
 	OutputLine(NULL, "　間違いなくレナだったよね？　ポスターにも、ジャケットにもピンで載ったくらいだし」",
-			NULL, " It was definitely Rena, right? That's why I'm on all the posters and covers.\"", Line_Normal);
+			NULL, " It was definitely Rena, right? That's why there're some posters and covers of just me alone.\"", Line_Normal);
```
Comment (DoctorDiablo) [inline Update/mio_omake_switch.txt:761]:
```suggestion
			NULL, " It was definitely Rena, right? That's why there are some posters and covers of just me alone.\"", Line_Normal);
```

File: Update/mio_omake_switch.txt [hunk 4]
```diff
@@ -3344,15 +3344,15 @@ void main()
 	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#8676cf>羽入</color>", NULL, "<color=#8676cf>Hanyuu</color>", NULL, Line_ContinueAfterTyping); }
 	ModPlayVoiceLS(4, 12, "switch/s20/12/ot_han_0050", 256, TRUE);
 	OutputLine(NULL, "「って魅音、さっき老けた自分と対面したショックからもう立ち直ったのですか？」",
-			NULL, "\"Hey Mion, have you recovered yet from the shock of meeting your future self?\"", Line_Normal);
+			NULL, "\"Hey Mion, have you recovered already from the shock of meeting your future self?\"", Line_Normal);
```
Comment (DoctorDiablo) [inline Update/mio_omake_switch.txt:3347]:
```suggestion
			NULL, "\"Hey Mion, have you recovered yet from the shock of meeting your future self?\"", Line_Normal);
```

File: Update/mio_omake_switch.txt [hunk 5]
```diff
@@ -3855,7 +3855,7 @@ void main()
 	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#ec7bb8>田村</color>", NULL, "<color=#ec7bb8>Tamura</color>", NULL, Line_ContinueAfterTyping); }
 	ModPlayVoiceLS(4, 51, "switch/s20/51/ot_tam_0020", 256, TRUE);
 	OutputLine(NULL, "「中の人などおらぬッッ！ 内輪の話は控え給え、角の民の巫女よ！！」",
-			NULL, "\"There are no such people! Do refrain from speaking of confidential matters, miko of the people of horns!!\"", Line_Normal);
+			NULL, "\"There are no such person! Do refrain from speaking of confidential matters, miko of the people of horns!!\"", Line_Normal);
```
Comment (DoctorDiablo) [inline Update/mio_omake_switch.txt:3858]:
```suggestion
			NULL, "\"There is no such person! Do refrain from speaking of confidential matters, miko of the people of horns!!\"", Line_Normal);
```

File: Update/mio_omake_switch.txt [hunk 6]
```diff
@@ -3956,14 +3956,14 @@ void main()
...
-			NULL, "\"...My existence... shouldn't be... buried in the shadows like this.\"", Line_Normal);
+			NULL, "\"...My existence... shouldn't be... conspicuous.\"", Line_Normal);
 	ClearMessage();
 
 //梨花rvS20/05/ot_rika_0220.「でしょうね。こんなキャラの強烈な神様に挟まれたんじゃ、おいそれと差し出がましく首を突っ込むわけにもいかないし」
 	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#6972c1>梨花</color>", NULL, "<color=#6972c1>Rika</color>", NULL, Line_ContinueAfterTyping); }
 	ModPlayVoiceLS(4, 5, "switch/s20/05/ot_rika_0220", 256, TRUE);
 	OutputLine(NULL, "「でしょうね。こんなキャラの強烈な神様に挟まれたんじゃ、おいそれと差し出がましく首を突っ込むわけにもいかないし」",
-			NULL, "\"Right? When you're standing between two powerful gods like this, you can't just go sticking your nose in things.\"", Line_Normal);
+			NULL, "\"I know it. When you're standing between two striking gods like this, you can't just go sticking your nose in things.\"", Line_Normal);
```
Comment (DoctorDiablo) [inline Update/mio_omake_switch.txt:3966]:
```suggestion
			NULL, "\"Right? When you're standing between two striking gods like this, you can't just go sticking your nose in things.\"", Line_Normal);
```

File: Update/mio_omake_switch.txt [hunk 7]
```diff
@@ -4138,7 +4138,7 @@ void main()
 	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#945c44>富竹</color>", NULL, "<color=#945c44>Tomitake</color>", NULL, Line_ContinueAfterTyping); }
 	ModPlayVoiceLS(4, 8, "ps3/s20/08/440800165", 256, TRUE);
 	OutputLine(NULL, "「自分にとって大事なものを守ることに、……僕は臆病でありすぎたのかもしれない。",
-			NULL, "\"Perhaps I was too cowardly... to protect what mattered to me.", Line_WaitForInput);
+			NULL, "\"...to confirm the truth and to protect what mattered to me.", Line_WaitForInput);
```
Comment (DoctorDiablo) [inline Update/mio_omake_switch.txt:4141]:
```suggestion
			NULL, "\"...To confirm the truth and to protect what mattered to me.", Line_WaitForInput);
```

File: Update/mio_omake_switch.txt [hunk 8]
```diff
@@ -4383,7 +4383,7 @@ void main()
 	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#6972c1>梨花</color>", NULL, "<color=#6972c1>Rika</color>", NULL, Line_ContinueAfterTyping); }
 	ModPlayVoiceLS(4, 5, "ps3/s20/05/440500206", 256, TRUE);
 	OutputLine(NULL, "「いま自分が持っている何かは独力で勝ち取ったものもあれば、誰かから与えられて得たものもあります。",
-			NULL, "\"If you have things now that you obtained out of your own strength, then there are also things that others gave to you.", Line_WaitForInput);
+			NULL, "\"Some things that you have now are obtained out of your own strength, and others are given to you from someone.", Line_WaitForInput);
```
Comment (DoctorDiablo) [inline Update/mio_omake_switch.txt:4386]:
```suggestion
			NULL, "\"Some things that you have now were obtained out of your own strength, and others were given to you by someone else.", Line_WaitForInput);
```

File: Update/mio_omake_ps3.txt [hunk 9]
```diff
@@ -136,7 +136,7 @@ void main()
 	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#5ec69a>詩音</color>", NULL, "<color=#5ec69a>Shion</color>", NULL, Line_ContinueAfterTyping); }
 	ModPlayVoiceLS(4, 6, "ps3/s20/06/440600185", 256, TRUE);
 	OutputLine(NULL, "まぁ、たくさん話に参加できたのに文句を言っちゃ、罰が当たるってもんですけどね」",
-			NULL, "Though I did spend most of them complaining despite playing a part, so I guess I got what I deserved.\"", Line_Normal);
+			NULL, "Though thankfully we had many chances to play a part, so we should not moan about it, you know.\"", Line_Normal);
```
Comment (DoctorDiablo) [inline Update/mio_omake_ps3.txt:139]:
```suggestion
			NULL, "Though thankfully we had many chances to play a part, so we shouldn't moan about it, you know.\"", Line_Normal);
```

File: Update/mio_omake_ps3.txt [hunk 10]
```diff
@@ -461,7 +461,7 @@ void main()
 	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#f0953d>レナ</color>", NULL, "<color=#f0953d>Rena</color>", NULL, Line_ContinueAfterTyping); }
 	ModPlayVoiceLS(4, 2, "ps3/s20/02/440200412", 256, TRUE);
 	OutputLine(NULL, "「……本当にそれ、自慢じゃないね」",
-			NULL, "\"...Should you really be bragging about that?\"", Line_Normal);
+			NULL, "\"...Indeed, it's really not a bragging.\"", Line_Normal);
```
Comment (DoctorDiablo) [inline Update/mio_omake_ps3.txt:464]:
```suggestion
			NULL, "\"...Indeed, that really isn't bragging.\"", Line_Normal);
```

File: Update/mio_omake_ps3.txt [hunk 11]
```diff
@@ -767,7 +767,7 @@ void main()
 			NULL, " Rena was the main heroine when \"Higurashi\" started, right?", Line_WaitForInput);
 	ModPlayVoiceLS(4, 2, "ps3/s20/02/440200418", 256, TRUE);
 	OutputLine(NULL, "　間違いなくレナだったよね？　ポスターにも、ジャケットにもピンで載ったくらいだし」",
-			NULL, " It was definitely Rena, right? That's why I'm on all the posters and covers.\"", Line_Normal);
+			NULL, " It was definitely Rena, right? That's why there're some posters and covers of just me alone.\"", Line_Normal);
```
Comment (DoctorDiablo) [inline Update/mio_omake_ps3.txt:770]:
```suggestion
			NULL, " It was definitely Rena, right? That's why there are some posters and covers of just me alone.\"", Line_Normal);
```

File: Update/mio_omake_ps3.txt [hunk 12]
```diff
@@ -3197,7 +3197,7 @@ void main()
 	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#945c44>富竹</color>", NULL, "<color=#945c44>Tomitake</color>", NULL, Line_ContinueAfterTyping); }
 	ModPlayVoiceLS(4, 8, "ps3/s20/08/440800165", 256, TRUE);
 	OutputLine(NULL, "「自分にとって大事なものを守ることに、……僕は臆病でありすぎたのかもしれない。",
-			NULL, "\"Perhaps I was too cowardly... to protect what mattered to me.", Line_WaitForInput);
+			NULL, "\"...to confirm the truth and to protect what mattered to me.", Line_WaitForInput);
```
Comment (DoctorDiablo) [inline Update/mio_omake_ps3.txt:3200]:
```suggestion
			NULL, "\"...To confirm the truth and to protect what mattered to me.", Line_WaitForInput);
```

File: Update/mio_omake_ps3.txt [hunk 13]
```diff
@@ -3442,7 +3442,7 @@ void main()
 	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#6972c1>梨花</color>", NULL, "<color=#6972c1>Rika</color>", NULL, Line_ContinueAfterTyping); }
 	ModPlayVoiceLS(4, 5, "ps3/s20/05/440500206", 256, TRUE);
 	OutputLine(NULL, "「いま自分が持っている何かは独力で勝ち取ったものもあれば、誰かから与えられて得たものもあります。",
-			NULL, "\"If you have things now that you obtained out of your own strength, then there are also things that others gave to you.", Line_WaitForInput);
+			NULL, "\"Some things that you have now are obtained out of your own strength, and others are given to you from someone.", Line_WaitForInput);
```
Comment (DoctorDiablo) [inline Update/mio_omake_ps3.txt:3445]:
```suggestion
			NULL, "\"Some things that you have now were obtained out of your own strength, and others were given to you by someone else.", Line_WaitForInput);
```

**Review by DoctorDiablo — APPROVED** [review]

**Review by DoctorDiablo — APPROVED** [review]

**Pull request merged**